### PR TITLE
CaaSP cluster migration from GM to current build

### PR DESCRIPTION
**Pull request by kravciak**
Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/935

Local runs:
 - http://dhcp126.suse.cz/tests/8493 - 3.0 -> 4.0

Update process:
 - set up cluster from CaaSP-GM
 - register during installation to get updates
 - deregister after installation, add extracted ISO as repository
 - run transactional-update dup and reboot from velum

Sample repo variables:
```
REPO_0=SUSE-CaaS-Platform-3.0-DVD-x86_64-Build0101-Media1
INCIDENT_REPO=http://openqa.suse.de/assets/repo/%REPO_0%
```

### CODE REVIEW

File: lib/caasp.pm [hunk 1]
```diff
@@ -81,19 +81,19 @@ sub rpmver {
...
-        script_run "journalctl > journal.log", 60;
-        upload_logs "journal.log";
-
-        script_run 'supportconfig -b -B supportconfig', 500;
-        upload_logs '/var/log/nts_supportconfig.tbz';
-
-        upload_logs('/var/log/transactional-update.log', failok => 1);
-        upload_logs('/var/log/YaST2/y2log-1.gz') if get_var 'AUTOYAST';
-    }
+    #    if (is_caasp 'local') {
```
Comment (rwx788) [inline lib/caasp.pm]: Please, remove commented code, we have git log in for history.

Comment (thehejik) [reply lib/caasp.pm]: It doesn't work for me. it seems that updates were not installed and velum doesn't show any node with avail upgrades. Still investigating whats wrong. http://dhcp138.suse.cz/tests/844#step/stack_update/14

**Comment by thehejik** [comment]
It doesn't work for me. it seems that updates were not installed and velum doesn't show any node with avail upgrades. Still investigating whats wrong. http://dhcp138.suse.cz/tests/844#step/stack_update/14

**Comment by kravciak** [comment]
@thehejik - sorry for confusion, I hacked it a little to do 2.0 -> 3.0 migration and some changes went to 3.0 -> 4.0 dup workflow.
It should be fixed now, I added needles now and doing new local runs.

**Pull request merged**